iFixit/react-commerce
# Rename /Community to /CommunityNext

## Overview
Currently, our nextjs store routes links to ifixit.com/Community to the nextjs version of /Community, which is not ready yet and does not show the header or footer. 

Simply renaming this directory in react-commerce will force nextjs to use an external link rather than finding a path in the repo.

## QA
- On a nextjs product page ([you can use this one!](http://localhost:3000/products/2-tb-ssd-hybrid-3-5-hard-drive)), use the header to navigate to the community page (header > Community > Get Involved)
- Check that the page is loaded with the header and footer

Closes: #1020
